conans/client/generators.py

```diff
@@ -66,7 +56,29 @@ def content(self):
                                             deps=deps)
             sections.append(dep_flags)
 
+        # GENERAL VARIABLES
+        deps = DepsCppCmake(self._build_info)
+
+        template = ('SET(CONAN_INCLUDE_DIRS {deps.include_paths} ${{CONAN_INCLUDE_DIRS}})\n'
+            'SET(CONAN_LIB_DIRS {deps.lib_paths} ${{CONAN_LIB_DIRS}})\n'
+            'SET(CONAN_BIN_DIRS {deps.bin_paths} ${{CONAN_BIN_DIRS}})\n'
+            'SET(CONAN_LIBS {deps.libs} ${{CONAN_LIBS}})\n'
+            'SET(CONAN_DEFINES {deps.defines} ${{CONAN_DEFINES}})\n'
+            'SET(CONAN_CXX_FLAGS "{deps.cppflags} ${{CONAN_CXX_FLAGS}}")\n'
+            'SET(CONAN_SHARED_LINK_FLAGS "{deps.sharedlinkflags} ${{CONAN_SHARED_LINK_FLAGS}}")\n'
+            'SET(CONAN_EXE_LINKER_FLAGS "{deps.exelinkflags} ${{CONAN_EXE_LINKER_FLAGS}}")\n'
+            'SET(CONAN_C_FLAGS "{deps.cflags} ${{CONAN_C_FLAGS}}")\n'
+            'SET(CONAN_CMAKE_MODULE_PATH {module_paths} ${{CONAN_CMAKE_MODULE_PATH}})')
+
+        rootpaths = [DepsCppCmake(dep_cpp_info).rootpath for _, dep_cpp_info
```

Comment:
Why the need to create a DepsCppCMake object, to obtain the rootpath? Cannot be obtained from dep_cpp_info directly? Seems because of the backslash issue on windows, right?